selfdrive/car/toyota/carcontroller.py

```diff
@@ -112,7 +112,7 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, hud_alert,
       lead = lead or CS.out.vEgo < 12.    # at low speed we always assume the lead is present do ACC can be engaged
 
       # Lexus IS uses a different cancellation message
-      if pcm_cancel_cmd and CS.CP.carFingerprint == CAR.LEXUS_IS:
+      if pcm_cancel_cmd and CS.CP.carFingerprint in [CAR.LEXUS_IS, CAR.LEXUS_ISH]:
```

Comment:
is this confirmed to work?

Reply:
This is usually used for the DM check failing out right?

Reply:
this is used any time openpilot disengages itself

Reply:
I'll work with @bisra2 to test this with the seatbelt unbuckled disengagement. 

I've actually never had OP disengage on me ever except due to kernel panics or some software/hardware issue so I think I'll try it myself on my own car first before guiding @bisra2 on this way.

Reply:
open an issue if you're hitting kernel panics, that shouldn't happen

Reply:
> open an issue if you're hitting kernel panics, that shouldn't happen

It happened a long time ago on mine, and many months ago. I'll keep that in mind to do if it happens again.

Reply:
@bisra2, I don't think the cruise control cancellation command is working here or at least openpilot isn't disengaging. When the seatbelt is unbuckled, there's no period of cruise control being disengaged afterwards. 

@eFiniLan , it's been a long time, but do you remember testing this functionality on your IS300H when you had it? 

I think we might need to figure out an alternate working command here. 

@bisra2's IS300H:

It goes Green, Red, and Green. The `CRUISE_ACTIVE` signal never goes low during the seatbelt unbuckling.

![image](https://user-images.githubusercontent.com/5363/119074808-8c9ff380-b9a4-11eb-8aee-42e72b9c2c4c.png)

![image](https://user-images.githubusercontent.com/5363/119075946-8ca0f300-b9a6-11eb-8924-fed5e7866562.png)


My 2020 Corolla:

It goes Green, Red, Blue, and Green as I turn back on openpilot after unbuckling the seatbelt. The `CRUISE_ACTIVE` signal goes low during the seatbelt unbuckling.

![image](https://user-images.githubusercontent.com/5363/119074838-97f31f00-b9a4-11eb-92ed-a52a83a57915.png)

![image](https://user-images.githubusercontent.com/5363/119075996-a4787700-b9a6-11eb-9bd6-66b93f473581.png)

Reply:
@nelsonjchen as far as I remembered that part never worked, OP itself disengaged but stock ACC remains.

Reply:
I've asked ash11111111#7507 about the OP disengage on gas on Discord who has a US/CAN IS300 but they are busy fighting some other hardware issue with their Comma Two at the moment to respond. ash11111111#7507 did say the stock system doesn't disengage on gas but we all know that.

Reply:
I can confirm the IS300 disengages on gas and works with the current cancellation message.

Reply:
The seatbelt unlatched event is a soft disable, meaning it doesn't disengage immediately. Gas pedal disengage would be the most direct way to test the cancellation. This is a blocker for merging.

Reply:
Ok, I'll put this back to draft for now. In the meantime, I will repeat what I received from @bisra2 after asking "To be clear: Pressing the gas pedal disengages openpilot, but the automatic cruise control continues to be enabled?":


Test 1: 
Openpilot active, unbuckling the seatbelt: The stock automatic cruise control continues to be enabled. The car slows down and accelerates on its own depending on the traffic but no more openpilot
Test2:
Openpilot activated, I press the gas pedal.:The stock automatic cruise control continues to be enabled. The car slows down and accelerates on its own depending on the traffic but no more openpilot
So the answer to your question is: Yes
To summarize, pressing the gas pedal or unbuckling the seat belt causes openpilot to disengage, but the stock safety sense continues to operate in both cases.